js/base/Exchange.js

```diff
@@ -2793,14 +2793,7 @@ module.exports = class Exchange {
          * @param {object} params extra parameters specific to the exchange api endpoint
          * @returns {[string|undefined, object]} the marginMode in lowercase as specified by params["marginMode"], params["defaultMarginMode"] this.options["marginMode"] or this.options["defaultMarginMode"]
          */
-        const defaultMarginMode = this.safeString2 (this.options, 'marginMode', 'defaultMarginMode');
-        const methodOptions = this.safeValue (this.options, methodName, {});
-        const methodMarginMode = this.safeString2 (methodOptions, 'marginMode', 'defaultMarginMode', defaultMarginMode);
-        const marginMode = this.safeStringLower2 (params, 'marginMode', 'defaultMarginMode', methodMarginMode);
-        if (marginMode !== undefined) {
-            params = this.omit (params, [ 'marginMode', 'defaultMarginMode' ]);
-        }
-        return [ marginMode, params ];
+        return this.handleOptionAndParams (params, methodName, 'marginMode');
```

Comment:
@ttodua shouldn't we also accept and route the defaultValue ?

Reply:
that's what I asked too in chat: https://discord.com/channels/690203284119617602/921046068555313202/1044699473575694437
we don't have any default for it. 
`handleOptionAndParams` does everything as this method did, but neither here we had any default value (should we set `isolated` as default? if no defaults needed, then we might also remove default from `handleSubTypeAndParams`?)